The package manager is undefined

**Description**

The project run in Expo Go (from test-apps folder) can't install dependencies. The package manager is `undefined`. 

**Environment**

- Radon IDE: 0.23.0
- Expo Go (from test-apps), Expo 50, RN 0.73.6

```

2024-11-25 09:09:42.952 [info] Found app root folder: /Users/maciejstosio/SWM/radon-ide/test-apps/expo-go
2024-11-25 09:09:43.470 [info] Launching devtools
2024-11-25 09:09:43.470 [info] Launching metro
2024-11-25 09:09:43.498 [info] Devtools started on port 52121
2024-11-25 09:09:44.292 [error] Error: Command failed with ENOENT: undefined --version
spawn undefined ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) Command undefined --version execution resulted in an error:
2024-11-25 09:09:44.292 [error] Required package manager: undefined is not installed
2024-11-25 09:09:44.293 [info] Installing node modules
2024-11-25 09:09:44.418 [error] Error: Command failed with ENOENT: undefined --version
spawn undefined ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) Command undefined --version execution resulted in an error:
2024-11-25 09:09:44.418 [error] Required package manager: undefined is not installed
2024-11-25 09:09:44.418 [info] Installing node modules succeeded
2024-11-25 09:09:44.419 [error] Error: Cannot find module 'react-native'
Require stack:
- /Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1232:15)
    at Function.n._resolveFilename (node:electron/js2c/utility_init:2:16068)
    at Function.t._resolveFilename (file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:166:22715)
    at Function.resolve (node:internal/modules/helpers:190:19)
    at Metro.launchPackager (/Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/src/project/metro.ts:157:15)
    at Metro.startInternal (/Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/src/project/metro.ts:211:29)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) Unhandled promise rejection
2024-11-25 09:09:44.419 [error] Error: Cannot find module 'react-native'
Require stack:
- /Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1232:15)
    at Function.n._resolveFilename (node:electron/js2c/utility_init:2:16068)
    at Function.t._resolveFilename (file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:166:22715)
    at Function.resolve (node:internal/modules/helpers:190:19)
    at Metro.launchPackager (/Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/src/project/metro.ts:157:15)
    at Metro.startInternal (/Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/src/project/metro.ts:211:29)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) Unhandled promise rejection
2024-11-25 09:09:44.569 [error] Error: Command failed with ENOENT: undefined --version
spawn undefined ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) Command undefined --version execution resulted in an error:
2024-11-25 09:09:44.569 [error] Required package manager: undefined is not installed
2024-11-25 09:09:44.570 [error] Error: Command failed with ENOENT: undefined --version
spawn undefined ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) Command undefined --version execution resulted in an error:
2024-11-25 09:09:44.570 [error] Required package manager: undefined is not installed
2024-11-25 09:09:44.606 [info] Could not find Xcode project files in "/Users/maciejstosio/SWM/radon-ide/test-apps/expo-go/ios" folder
2024-11-25 09:09:44.610 [info] Could not find Xcode project files in "/Users/maciejstosio/SWM/radon-ide/test-apps/expo-go/ios" folder
2024-11-25 09:09:44.619 [info] Selected device is ready
2024-11-25 09:09:44.619 [error] Error: Cannot find module 'react-native'
Require stack:
- /Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1232:15)
    at Function.n._resolveFilename (node:electron/js2c/utility_init:2:16068)
    at Function.t._resolveFilename (file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:166:22715)
    at Function.resolve (node:internal/modules/helpers:190:19)
    at Metro.launchPackager (/Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/src/project/metro.ts:157:15)
    at Metro.startInternal (/Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/src/project/metro.ts:211:29)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) Couldn't start device session
2024-11-25 09:12:07.391 [info] Selected device is ready
2024-11-25 09:12:07.391 [error] Error: Cannot find module 'react-native'
Require stack:
- /Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/dist/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1232:15)
    at Function.n._resolveFilename (node:electron/js2c/utility_init:2:16068)
    at Function.t._resolveFilename (file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:166:22715)
    at Function.resolve (node:internal/modules/helpers:190:19)
    at Metro.launchPackager (/Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/src/project/metro.ts:157:15)
    at Metro.startInternal (/Users/maciejstosio/SWM/radon-ide/packages/vscode-extension/src/project/metro.ts:211:29)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) Couldn't start device session

```

--- a/packages/vscode-extension/src/utilities/packageManager.ts
+++ b/packages/vscode-extension/src/utilities/packageManager.ts
@@ -88,7 +88,7 @@ export async function resolvePackageManager(): Promise<PackageManagerInfo | unde
     for (const file of files) {
       const manager = lockFiles.get(file);
       if (manager) {
-        packageManagerCandidates.push(packageManager);
+        packageManagerCandidates.push(manager);
       }
     }
 

